README.md

```diff
@@ -235,3 +235,13 @@ Interaction with GNU `make` jobserver
 sccache provides support for a [GNU make jobserver](https://www.gnu.org/software/make/manual/html_node/Job-Slots.html). When the server is started from a process that provides a jobserver, sccache will use that jobserver and provide it to any processes it spawns. (If you are running sccache from a GNU make recipe, you will need to prefix the command with `+` to get this behavior.) If the sccache server is started without a jobserver present it will create its own with the number of slots equal to the number of available CPU cores.
 
 This is most useful when using sccache for Rust compilation, as rustc supports using a jobserver for parallel codegen, so this ensures that rustc will not overwhelm the system with codegen tasks. Cargo implements its own jobserver ([see the information on `NUM_JOBS` in the cargo documentation](https://doc.rust-lang.org/stable/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts)) for rustc to use, so using sccache for Rust compilation in cargo via `RUSTC_WRAPPER` should do the right thing automatically.
+
+---
+
+Known Caveats
+-------------
+
+* `proc-macro` crates like `serde_derive` cannot be cached.
+* `bin` crates cannot be cached.
+* Incrementally compiled crates cannot be cached. First party crates are compiled incrementally when building in debug mode. You may find that release builds are faster than debug builds because of this.
```

Comment:
It would be useful to mention setting the environment variable `CARGO_INCREMENTAL=0` to disable incremental compilation.

Reply:
I'm not sure I'd want to directly recommend this here, TBH. It'd probably be best to just state the facts and link to [the cargo documentation](https://doc.rust-lang.org/cargo/reference/profiles.html#incremental). "sccache cannot cache incrementally compiled crates. By default, in the debug profile, cargo will use incremental compilation for workspace members and path dependencies."